verify-c-common.Dockerfile

```diff
@@ -0,0 +1,27 @@
+
+FROM seahorn/seahorn-llvm10:nightly
+
+ENV SEAHORN=/home/usea/seahorn/bin/sea PATH="$PATH:/home/usea/seahorn/bin:/home/usea/bin"
+
+## install required pacakges
+USER root
+
+## clone verify-c-common repository
+USER usea
+WORKDIR /home/usea
+RUN git clone https://github.com/yvizel/verify-c-common.git 
+
+## clone trusty repository (takes a long time)
+WORKDIR /home/usea/verify-c-common
+RUN git clone https://github.com/awslabs/aws-c-common.git
+
+WORKDIR /home/usea/verify-c-common/aws-c-common
+RUN mkdir build && cd build && cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ../
+
+WORKDIR /home/usea/verify-c-common
+RUN ln -sf aws-c-common/build/compile_commands.json .
```

Comment:
remove above `RUN`. it is no longer necessary.

Reply:
are you referring to line 19 or 22?

Reply:
line 22